src/main/kotlin/app/revanced/patches/youtube/misc/integrations/IntegrationsPatch.kt

```diff
@@ -1,15 +1,13 @@
-package app.revanced.patches.youtube.misc.integrations.patch
+package app.revanced.patches.youtube.misc.integrations
 
-import app.revanced.patcher.annotation.Name
-import app.revanced.patcher.patch.annotations.RequiresIntegrations
-import app.revanced.patches.shared.integrations.patch.AbstractIntegrationsPatch
+import app.revanced.patcher.patch.annotation.Patch
+import app.revanced.patches.shared.integrations.AbstractIntegrationsPatch
 import app.revanced.patches.youtube.misc.integrations.fingerprints.*
 
-@Name("Integrations")
-@RequiresIntegrations
-class IntegrationsPatch : AbstractIntegrationsPatch(
+@Patch(requiresIntegrations = true)
```

Comment:
❓ : Missing patch name

```suggestion
@Patch(
    name = "Integrations"
    requiresIntegrations = true
)
```